public/app/features/explore/RichHistory/RichHistorySettingsTab.tsx

```diff
@@ -78,7 +78,6 @@ export function RichHistorySettingsTab(props: RichHistorySettingsProps) {
         <Field
           label="History time span"
           description={`Select the period of time for which Grafana will save your query history. Up to ${MAX_HISTORY_ITEMS} entries will be stored.`}
-          className="space-between"
```

Comment:
was the removal of this intentional or did you forget 
```
className={styles.spaceBetween}
```

Reply:
I checked whether it's needed and it looked the same after I disabled it. That's why I decided to delete it.

Reply:
notice the gap between the select and the `Change the default active tab from “Query history” to “Starred"` toggle. 

after:
<img width="462" alt="Screenshot 2022-07-22 at 16 58 02" src="https://user-images.githubusercontent.com/1170767/180478180-9a4300b3-2b61-41a6-81e1-07953e3398b9.png">
before:
<img width="536" alt="Screenshot 2022-07-22 at 16 58 18" src="https://user-images.githubusercontent.com/1170767/180478184-aae3aa0c-d9a1-469a-b89e-f6a69a9f42e7.png">

looks ok to me, but it's not the same